### DISCUSSION TITLE
Why GitHub asking for payment even I have active student developer pack.

### DISCUSSION THREAD

**Discussion by MohitGupta121** [community/community#33726] [GitHub Education]
<img width="1249" alt="Screenshot 2022-09-21 at 10 48 36 PM" src="https://user-images.githubusercontent.com/76530270/191569503-68cad1a6-7a63-4574-88b5-6654664ff9b5.png">

Why I'm getting this payment overdue even though I'm a student and have an active student developer pack?

<img width="1356" alt="Screenshot 2022-09-21 at 10 50 55 PM" src="https://user-images.githubusercontent.com/76530270/191569934-5944e453-4255-421b-abd1-aeeff762d8ec.png">

**Comment by riyadhuddin** [comment]
do you have an organization account?

**Reply by MohitGupta121** [reply]
No

**Comment by samus-aran** [comment]
Hi @MohitGupta121, this may not be related to the Student Developer Pack. Have you got any other services on your GitHub account? Perhaps [Actions](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions), or have a look at your [subscriptions](https://docs.github.com/en/billing/managing-your-github-billing-settings/viewing-your-subscriptions-and-billing-date).

**Reply by MohitGupta121** [reply]
no nothing, only recently I got sponsor tab.

**Reply by riyadhuddin** [reply]
if you don't wish to sponsor someone just cancel it, otherwise you will have to pay.

**Reply by MohitGupta121** [reply]
> if you don't wish to sponsor someone just cancel it, otherwise you will have to pay.

I sponsor only one time. And that already paid successfully.
